src/orion/algo/tpe.py

```diff
@@ -266,89 +270,130 @@ def set_state(self, state_dict):
         self.seed_rng(state_dict["seed"])
         self.rng.set_state(state_dict["rng_state"])
 
-    def suggest(self, num=1):
+    def _n_suggested(self):
```

Comment:
Seems no difference compared with the `n_suggested` property in base algorithm?

Reply:
Oh, right, thanks for catching this! I will remove it.

Reply:
Oh, right, thanks for catching this! I will remove it.